Oqtane.Test/Oqtane.Shared.Tests/UtilitiesTests.cs

```diff
@@ -8,13 +8,13 @@ public class UtilitiesTests
         [Theory]
         [InlineData("contoso", "login", "returnUrl=/admin", "/contoso/login?returnUrl=/admin")]
         [InlineData("contoso", "admin", "", "/contoso/admin")]
-        [InlineData("contoso", "", "pageId=4", "/contoso?pageId=4")]
-        [InlineData("contoso", "", "pageId=4&moduleId=10", "/contoso?pageId=4&moduleId=10")]
+        [InlineData("contoso", "", "pageId=4", "/contoso/?pageId=4")]
```

Comment:
> When I debug the tests I see these 4 are failing:

    [InlineData("", "", "pageId=4", "?pageId=4")]
    [InlineData("contoso", "", "pageId=4&moduleId=10", "/contoso?pageId=4&moduleId=10")]
    [InlineData("", "", "pageId=4&moduleId=10", "?pageId=4&moduleId=10")]
    [InlineData("contoso", "", "pageId=4", "/contoso?pageId=4")]

> the results should be:

    [InlineData("", "", "pageId=4", "/?pageId=4")]
    [InlineData("contoso", "", "pageId=4&moduleId=10", "/contoso/?pageId=4&moduleId=10")]
    [InlineData("", "", "pageId=4&moduleId=10", "/?pageId=4&moduleId=10")]
    [InlineData("contoso", "", "pageId=4", "/contoso/?pageId=4")]

This is exactly what I did!!